tasks/messages/message_test.go

```diff
@@ -48,7 +48,7 @@ func TestParseMessageParams(t *testing.T) {
 			method:      2,
 			params:      mustDecodeBase64(t, "gm5maWwvMi9tdWx0aXNpZ1hFhVUBVZOiUErA4tz2x5Zz9iV/aBZBdHxVARJCEmhAYnLXQYa/1IScyah3BRgEVQE3/ahkH/xoK6Xpa0ZbDxkY/oPxiAIA"),
 			actorCode:   builtin5.InitActorCodeID,
-			wantMethod:  "InitExecParams",
+			wantMethod:  "Exec",
```

Comment:
The actual method name is just `Exec`: https://github.com/filecoin-project/specs-actors/blob/v6.0.1/actors/builtin/init/init_actor.go#L23